config/rbac/role.yaml

```diff
@@ -6,6 +6,25 @@ metadata:
   name: manager-role
 rules:
 - apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
+- apiGroups:
+  - ""
+  resources:
+  - endpoints
+  - services
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
   - ""
   resources:
   - endpoints
```

Comment:
The endpoint and services are repeated resources in this yaml, I doubt if these were generated by the `make manifests` command. Were these changes manually? 
If so, can you please run `make generate manifests` and see if the resources are repeated?